docs/en/siem/detection-engine-intro.asciidoc

```diff
@@ -93,9 +96,39 @@ To open and close signals, either:
 *Close/Open selected*.
 
 [float]
+[[signals-to-timelines]]
 === Send signals to the Timeline
```

Comment:
Can we use "Investigate signals in Timeline" rather than "Send". (Send implies that you might be moving it to the timeline, but it will actually stay in the signals table).